app/views/applications/show.html.erb

```diff
@@ -24,7 +25,13 @@
 
   <% if params[:search] %>
     <% @found_pets.each do |found_pet| %>
-      <li><%= found_pet.name %></li>
+      <li><%= found_pet.name %><%= form_with url: "/pet_applications", method: :post, local: true do |form|%>
+        <%= form.hidden_field :application_id, value: @application.id %>
```

Comment:
Glad we changed it from a patch to a post because we are making a new record that associates an attribute from that new record and updates the original form.